app/models/reservation.rb

```diff
@@ -282,6 +282,10 @@ def requires_but_missing_actuals?
   end
   alias problem? requires_but_missing_actuals?
```

Comment:
If we're only ever using the key or checking its presence, should we get rid of `time_data#problem?`?